modules/spo-container-profile-instances.adoc

```diff
@@ -73,28 +80,28 @@ spec:
 +
 [source,terminal]
 ----
-$ oc delete deployment nginx-deploy
+$ oc delete deployment nginx-deploy -n my-namespace
 ----
 
 . To merge the profiles, delete the profile recording by running the following command:
 +
 [source,terminal]
 ----
-$ oc delete profilerecording test-recording
+$ oc delete profilerecording test-recording -n my-namespace
 ----
 
 . To start the merge operation and generate the results profile, run the following command:
 +
 [source,terminal,subs="attributes+"]
 ----
-$ oc get {object} -lspo.x-k8s.io/recording-id=test-recording
+$ oc get {object} -lspo.x-k8s.io/recording-id=test-recording -n my-namespace
 ----
 +
 .Example output
 [source,terminal]
 ----
-NAME                          USAGE                                         STATE
-test-recording-nginx-record   test-recording-nginx-record_mytest1.process   Installed
+NAME                          STATUS       AGE
+test-recording-nginx-record   Installed    55s
```

Comment:
When  updating the test result for seccompprofile, please do not update the output for selinuxprofile in step 6 of https://62471--docspreview.netlify.app/openshift-enterprise/latest/security/security_profiles_operator/spo-selinux#spo-container-profile-instances_spo-selinux 

For selinuxprofile, please keep  it as:
```
NAME                          USAGE                                         STATE
test-recording-nginx-record   test-recording-nginx-record_mytest1.process   Installed
```

Reply:
@xiaojiey I've created two example output sections - I believe this is what you were asking for, please verify?

Reply:
@xiaojiey I understand now what you mean, thank you so much for the clarification.

Reply:
@xiaojiey this is done, thank you!